javascript/javascript2/week1/currencyConverter.js

```diff
@@ -0,0 +1,82 @@
+// currency-converter-script.js
```

Comment:
Is it really the name of this file?

Reply:
Nope, this is what I forget to remove when pushing in GIT. 
The name of file is currencyConverter.js